Source/Core/DolphinQt2/Config/SettingsWindow.cpp

```diff
@@ -11,14 +11,19 @@
 #include <QVBoxLayout>
 
 #include "DolphinQt2/Config/SettingsWindow.h"
+#include "DolphinQt2/MainWindow.h"
 #include "DolphinQt2/Resources.h"
 #include "DolphinQt2/Settings.h"
+#include "DolphinQt2/Settings/AudioPane.h"
 #include "DolphinQt2/Settings/GeneralPane.h"
 #include "DolphinQt2/Settings/InterfacePane.h"
 #include "DolphinQt2/Settings/PathPane.h"
 
-SettingsWindow::SettingsWindow(QWidget* parent) : QDialog(parent)
+SettingsWindow::SettingsWindow(MainWindow* parent) : QDialog(parent)
```

Comment:
It would probably better in this case yeah. But that kinda overreaches the goal of this PR. Will still move out the connect code to SettingsWindow though.